kubernetes_admission/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "6d009536-3383-4071-b45e-7771e0970cf4",
+  "app_id": "kubernetes-admission",
+  "display_on_public_website": false,
```

Comment:
If you want the docs to be live when this is merged, this need to be true.

Reply:
No, as of now we would like to not display this integration as we consider it to be embedded into the Datadog Cluster Agent.